CONTRIBUTING.md

```diff
@@ -83,7 +83,30 @@ Flutter tests use [package:flutter_test](https://github.com/flutter/flutter/tree
  * `cd packages/flutter_tools`
  * `dart test/all.dart`
 
-`flutter test --flutter-repo` is a shortcut for those working on the flutter repository itself which runs all tests inside the `flutter` package regardless of the current working directory.
+Those working on the Flutter repo might find it useful to have a command-line alias for running Flutter's own tests
+regardless of the current working directory. For example, in your `.bashrc` on Linux, or `.profile` on Mac, define:
+
+```sh
+# Where you cloned the flutter repo
+export FLUTTER_REPO_DIR=/path/to/flutter/repository
```

Comment:
Because this alias is defined in `.bashrc` or `.profile` this variable will be globally visible, and therefore you won't be able to have both a checkout of the repo _and_ a normal installation of Flutter (e.g. from the alpha branch). If we use `FLUTTER_ROOT` it will cause all Flutter installations to point to the same location, which might result in surprising behavior.

`FLUTTER_REPO_DIR` would only point to a _working_ checkout of Flutter, and it would not affect other installations of it.

I'm fine with switching over to `FLUTTER_ROOT` if the alternative behavior is what we actually want.